bin/deseq2_qc.r

```diff
@@ -102,10 +103,10 @@ if (min(dim(count.table))<=1)  { # No point if only one sample, or one gene
 }
 if (!opt$vst) {
     vst_name <- "rlog"
-    rld      <- rlog(dds)
+    rld      <- rlog(dds, blind=TRUE)   # blind=TRUE is the default and already implied by design=~1
```

Comment:
Sorry, kind of see where you're going. But if we re-state all the defaults we'll make the code unnecessarily complicated. 

```suggestion
    rld      <- rlog(dds, blind=TRUE) 
```